dotCMS/src/main/java/com/dotmarketing/portlets/htmlpageasset/business/render/page/PageView.java

```diff
@@ -196,7 +192,7 @@ public Contentlet getUrlContent() {
         private  Template template;
         // The map of Containers and their respective relationships with Content Types
         private  Collection<? extends ContainerRaw> containers;
-        private  HTMLPageAssetInfo page;
+        private  HTMLPageAsset page;
```

Comment:
Codacy found an issue: [Field page has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5685344)